public/docs/ts/latest/glossary.jade

```diff
@@ -54,7 +54,7 @@ include _util-fns
     The barrel itself is a module file that re-exports *selected* exports of other modules.
     
     Imagine three modules in a `heroes` folder:
-  code-example(format='').
+  code-example.
```

Comment:
Why removing `(format='')` here and elsewhere in this file?
Not against it. Just wondering why the change.